src/internal/cluster/zarf.go

```diff
@@ -98,3 +99,30 @@ func (c *Cluster) RecordPackageDeployment(pkg types.ZarfPackage, components []ty
 
 	c.Kube.CreateOrUpdateSecret(deployedPackageSecret)
 }
+
+// EnableRegistryHPAScaleDown enables or disables the HPA scale down for the Zarf Registry.
+func (c *Cluster) EnableRegistryHPAScaleDown(enable bool) error {
```

Comment:
Nit; Having the function being named `EnableRegistryHPAScaleDown()` but then accepting a Boolean that affects if HPA is scaled down or not seems like an antipattern (or at least a bad name 😛)

Thoughts on changing the function header to something along the liens of `ConfigureRegistryHPAScaleDown(policy v2beta2.ScalingPolicySelect)` ?

Reply:
@jeff-mccoy and I don't like the word "toggle".  That implies that it remembers state and flips back and forth when you call it.  I would prefer Set or Configure personally

Reply:
@Racer159 I think toggle works great as most of the time you want it enabled to ensure a single core isn't getting hammered. Seeing that the issue originally reported by @corang shows that problem was an edge case for a specific deployment originally. Seeing the scope (as I understand it) is to temporarily change the behavior of HPA when deploying large packages, the word `toggle` seems accurate.

Reply:
looked at different strategies and ended up just doing to diff functions. Not as DRY, but more readable I suppose...

`EnableRegHPAScaleDown()`
`DisableRegHPAScaleDown()`

![Screenshot 2023-02-03 at 12 12 07 PM](https://user-images.githubusercontent.com/882485/216676580-ca2714ba-6f19-45ff-8254-8f9993e15cc4.png)